tbapy/models.py

```diff
@@ -0,0 +1,178 @@
+class Team:
+    def __init__(self, json):
+        self.json = json
+        self.address = json['address']
+        self.city = json['city']
+        self.country = json['country']
+        self.gmaps_place_id = json['gmaps_place_id']
+        self.gmaps_url = json['gmaps_url']
+        self.home_championship = json['home_championship']
+        self.key = json['key']
+        self.lat = json['lat']
+        self.lng = json['lng']
+        self.location_name = json['location_name']
+        self.motto = json['motto']
+        self.name = json['name']
+        self.nickname = json['nickname']
+        self.postal_code = json['postal_code']
+        self.rookie_year = json['rookie_year']
+        self.state_prov = json['state_prov']
+        self.team_number = json['team_number']
+        self.website = json['website']
+
+
+class Event:
+    def __init__(self, json):
+        self.json = json
```

Comment:
I think I'll leave it like this to make it easier when I need to make it more complicated in the future.